lib/ui/views/projects/project_details_view.dart

```diff
@@ -156,25 +156,27 @@ class _ProjectDetailsViewState extends State<ProjectDetailsView> {
   }
 
   Widget _buildProjectDescription() {
-    return Padding(
-      padding: const EdgeInsets.symmetric(vertical: 4.0),
-      child: Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-          Text(
-            'Description :',
-            style: Theme.of(context).textTheme.headline6.copyWith(
-                  fontWeight: FontWeight.bold,
-                  fontSize: 18,
+    return (_recievedProject.attributes.description != '')
```

Comment:
`_recievedProject.attributes.description` -> `(_recievedProject.attributes.description ?? '')` would take care of the null check too.